File: easy/string_integer_is_balanced.py
def string_integer_is_balanced(num: str) -> bool:
    even, odd = 0, 0
    for i, val in enumerate(num):
        digit = int(val)
        if i % 2 == 0:
            even += digit
        else:
            odd += digit
    return even == odd

File: easy/string_integer_is_balanced.go
package easy

func StringIntegerIsBalanced(num string) bool {
	even := 0
	odd := 0
	for i, r := range num {
		digit := int(r) - int('0')
		if i%2 == 0 {
			even += digit
		} else {
			odd += digit
		}
	}
	return even == odd
}
